packages/gatsby-page-utils/src/ignore-path.ts

```diff
@@ -0,0 +1,38 @@
+import * as mm from "micromatch"
+
+interface IPathIgnoreOptions {
+  patterns: string | ReadonlyArray<string>
+  options?: mm.Options
+}
+
+export function ignorePath(
+  path: string,
+  ignore?: IPathIgnoreOptions | string | string[] | {} | null
```

Comment:
If you change `IPathIgnoreOptions` to have an optional `pattern` field, then you can probably just drop `{}` from the type of `ignore` here.

Reply:
Done